OWNERS

```diff
@@ -2,14 +2,10 @@ approvers:
 - evankanderson
 - grantr
 - Harwayne
-- vaikas-google
-- n3wscott
-- mattmoor
 - nachocano
 
 # Reviewers are suggested from the reviewers list first, then the approvers
 # list. To add reviewers while spreading the load among existing approvers,
 # copy the approvers to the reviewers list too.
 reviewers:
-- n3wscott
```

Comment:
can you add the same set of approvers here?

Reply:
done